packages/runtimes/js/src/loaders/worker/js-loader.js

```diff
@@ -0,0 +1,13 @@
+/* global importScripts:readonly*/
+const cacheLoader = require('../../cacheLoader');
+
+module.exports = cacheLoader(function loadJSBundle(bundle) {
+  return new Promise(function(resolve, reject) {
+    try {
+      importScripts(bundle);
```

Comment:
Could we change the loader API to accept multiple bundles? I believe `importScripts` blocks execution, but can take variadic arguments (`importScripts(bundle1, bundle2, bundle3)`), so we could download them in parallel.

Reply:
Done